source/brailleDisplayDrivers/albatross/threads.py

```diff
@@ -0,0 +1,165 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2022 NV Access Limited, Burman's Computer and Education Ltd.
+
+"""Threads for Tivomatic Caiku Albatross braille display driver.
+Classes:
+- L{ReadThread}
+- L{RepeatedTimer}
+"""
+
+import ctypes
+import serial
+
+from ctypes import byref
+from ctypes.wintypes import DWORD
+from logHandler import log
+from serial.win32 import (
+	ERROR_IO_PENDING,
+	EV_RXCHAR,
+	GetLastError,
+	SetCommMask
+)
+from threading import (
+	Event,
+	Thread,
+	Timer
+)
+from typing import Callable
+
+import api
+
+from IAccessibleHandler import SecureDesktopNVDAObject
+
+from .constants import KC_INTERVAL
+
+
+class ReadThread(Thread):
+	"""Controls most of read operations and tries to reconnect when needed."""
+
+	def __init__(
+			self,
+			readFunction: Callable,
+			disableFunction: Callable,

```

Comment:
What are the parameter types?

```suggestion
			self,
			readFunction: Callable[[],],
			disableFunction: Callable[[],],
```